src/main/java/seedu/address/model/person/ID.java

```diff
@@ -26,16 +26,20 @@ public ID(String id) {
     value = id;
   }
 
+  public ID() {
+    value = "";
+  }
+
   /**
    * Returns true if a given string is a valid id number.
    */
   public static boolean isValidId(String test) {
-    return test.matches(VALIDATION_REGEX);
+    return test.matches(VALIDATION_REGEX) || test.equals("");
```

Comment:
empty line is valid is? seems strange

Reply:
Because a finance can have empty ID if it is not relevant